uReplicator-Manager/pom.xml-org

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
remove this file

Reply:
It was removed in my commits, but not sure why this one still shows up in diff